src/borrow_check.md

```diff
@@ -8,7 +8,7 @@ enforcing a number of properties:
 - That you can't move a value while it is borrowed.
 - That you can't access a place while it is mutably borrowed (except through
   the reference).
-- That you can't mutate a place while it is shared borrowed.
+- That you can't mutate a place while it is shared or borrowed.
```

Comment:
@spastorino thanks for your careful eye... the line just prior does talk about "mutably borrowed". I know @nikomatsakis [likes](http://smallcultfollowing.com/babysteps/blog/2014/05/13/focusing-on-ownership) "unique borrow" (instead of "mutable borrow"), and it's opposite is "shared borrow" (instead of "immutable borrow"), and maybe that's how "shared" sneaked in.